doc/readme.html

```diff
@@ -316,18 +316,41 @@
       so, invoke <code>cmake</code> in the build instructions above as follows:
     </p>
 
-<pre>
-  cmake -DDEAL_II_COMPONENT_DOCUMENTATION=ON -DCMAKE_INSTALL_PREFIX=/path/install/dir ../deal.II
-</pre>
-
-<p>
-  For this to succeed, you will need <a href="http://www.perl.org/"
-					target="_top">Perl 5.x</a>,
-  <a href="http://www.doxygen.org/" target="_top">doxygen</a>
-  and <code>dot</code> (which is part of
-  the <a href="http://www.graphviz.org" target="_top">GraphViz</a>
-  package) to be installed.
-</p>
+    <pre>
+      cmake -DDEAL_II_COMPONENT_DOCUMENTATION=ON -DCMAKE_INSTALL_PREFIX=/path/install/dir ../deal.II
+    </pre>
+    
+    <p>
+      For this to succeed, you will need <a href="http://www.perl.org/"
+					    target="_top">Perl 5.x</a>,
+      <a href="http://www.doxygen.org/" target="_top">doxygen</a>
+      and <code>dot</code> (which is part of
+      the <a href="http://www.graphviz.org" target="_top">GraphViz</a>
+      package) to be installed.
+    </p>
+  
+    <p>
+      The documentation contains links to pictures (e.g., for the
+      tutorial programs) that are by default stored online at the
+      dealii.org domain. If you want to use the documentation
+      completely offline, you can run
+      the <code>contrib/utilities/makeofflinedoc.sh</code> script in
+      an installed documentation directory to download all images.
+    </p>
+    
+    <p> Finally, the default for locally installed documentation is to
+      render formulas as images. You can force formulas to be
+      displayed via the MathJax system by
+      adding <code>-DDEAL_II_DOXYGEN_USE_MATHJAX=ON</code> to the
+      CMake call above. These formulas are then rendered natively by
+      the
+      browser. With <code>-DDEAL_II_DOXYGEN_USE_ONLINE_MATHJAX=OFF<code>
+      CMake will try to find a local installation of MathJax scripts,
+      otherwise the online version of the scripts will be used in the
+      documentation. Note, if you are going to host a public copy of
+      deal.II documentation it is a good idea to check that online
+      version of MathJax scripts is actually used.
```

Comment:
Not necessarily. Maybe just remove the last sentence or rewrite as
"If you are going to host a public copy of the documentation the local MathJax location needs to be accessible by the client."